book/invalidation.md

```diff
@@ -737,9 +762,11 @@ class BlockLayout:
 However, recall that with dirty flags we must always think about
 setting them, checking them, and resetting them. The `get` method
 automatically checks the dirty flags, and the `set` method
-automatically resets them, but who *sets* the `zoom` dirty flags?
+automatically resets them, but who *marks* the `zoom` dirty flags?
+(Without marking them when they change, we won't be able to avoid
+layout work.)
```

Comment:
Is this really right? I think if you fail to mark them, you'll skip _too much_ work.

Reply:
Clarified